packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1103,6 +1104,26 @@ contract VotingV2 is
         return spamDeletionProposals[spamDeletionRequestId];
     }
 
+    /****************************************
+     *      MIGRATION SUPPORT FUNCTIONS     *
+     ****************************************/
+
+    /**
+     * @notice Function to enable retrieval of rewards on a previously migrated away from voting contract.
+     * @param votingContract the previous version of the voting contract.
+     * @param voterAddress voter for which rewards will be retrieved. Does not have to be the caller.
+     * @param roundId the round from which voting rewards will be retrieved from.
+     * @param toRetrieve array of PendingRequests which rewards are retrieved from.
+     */
+    function retrieveRewardsOnMigratedVotingContract(
+        address votingContract,
```

Comment:
Maybe we should have this value set in the contract rather than letting the caller pass it in. Just seems slightly safer to not allow a user to have the ability to force the voting contract to make an outbound call to any contract (despite the function selector being pre-specified). We've gotten this feedback before from OZ.

Reply:
seeing this _only_ applies for this upgrade (v1->v2) we could just hard code it inline with this call.

Reply:
immutable variable?

Reply:
I agree an immutable variable makes sense